### ISSUE TITLE
Sonoma bug

### ISSUE THREAD

**Issue by ofirgalcon** [tuxudo/icloud#8]
I see this on some Sonoma clients

```Nov 09 2023 19:55:25 +0000 Running icloud
Nov 09 2023 19:55:26 +0000 icloud Error: Traceback (most recent call last):
File "/usr/local/munkireport/scripts/icloud", line 215, in
main()
File "/usr/local/munkireport/scripts/icloud", line 206, in main
result = process_prefs(prefs)
File "/usr/local/munkireport/scripts/icloud", line 83, in process_prefs
preffile.update(get_extra_prefs(pref))
File "/usr/local/munkireport/scripts/icloud", line 139, in get_extra_prefs
pl = FoundationPlist.readPlist(photospath)
File "/usr/local/munkireport/munkilib/FoundationPlist.py", line 85, in readPlist
raise NSPropertyListSerializationException(errmsg)
munkilib.FoundationPlist.NSPropertyListSerializationException: b'stream had too few bytes' in file /Users/username/Library/Containers/com.apple.CloudPhotosConfiguration/Data/Library/Preferences/com.apple.CloudPhotosConfiguration.plist
Nov 09 2023 19:55:26 +0000 WARNING: icloud return code: 1```

**Comment by ofirgalcon** [comment]
Here's the relevant plist:

```defaults read ~/Library/Containers/com.apple.CloudPhotosConfiguration/Data/Library/Preferences/com.apple.CloudPhotosConfiguration.plist
{
    "com.apple.photo.icloud.cloudphoto" = 1;
    "com.apple.photo.icloud.myphotostream" = 0;
    "com.apple.photo.icloud.sharedstreams" = 1;
    "service-preferences-com.apple.photo.icloud.cloudphoto" =     {
        CPSPreferenceOnlyKeepThumbnailsKey = 1;
    };
}

**Comment by ofirgalcon** [comment]
I think this is a permission issue. If I run the icloud script from Warp (terminal alt) i get a popup saying it needs permissions. 

<img width="372" alt="image" src="https://github.com/tuxudo/icloud/assets/16665880/4e9ba728-c9a3-4100-aa87-b13824a43738">

**Comment by tuxudo** [comment]
This should be fixed with version v3.1 of the module
